cookieConsent/OneTrust/index.js

```diff
@@ -0,0 +1,3 @@
+import OneTrust from "./browser";
+
+export default OneTrust;
```

Comment:
Convert this to a named export. Not absolutely necessary now but useful for plugin model.

Reply:
done